app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryUiController.kt

```diff
@@ -21,6 +21,10 @@ class MedicalHistorySummaryUiController(
 ) : ObservableTransformer<UiEvent, UiChange> {
 
   override fun apply(events: Observable<UiEvent>): ObservableSource<UiChange> {
+    return displayMedicalHistory(events)
```

Comment:
This commit is actually reordered to be before the previous one. This is not reflected in the PR since it was force-pushed later.